src/lib.rs

```diff
@@ -360,6 +369,28 @@ pub struct PostTextChange<BlockNumber, Moment> {
     text: Vec<u8>,
 }
 
+bitflags! {
```

Comment:
Yes, PostReaction is a type of reaction to a post. But the reaction maybe the combination of each different type of reaction. For example, you may like a post, also thumb up it. I design it as a bitflag as it can expand and accept much more type of reactions in the future.

Reply:
By expand, this would still require a runtime upgrade? so what we are saving, compared to having an explicit enum, is just that we do not need to run migration logic during a runtime upgrade?

If the answer is Yes, then I think we still should stick to the clearest way. Our runtime has so many places where runtime upgrades will require migrations anyway, so that ship has already sailed for us really, no need sacrificing clarity to _possibly_ about it in this one case. 

Also, I do not actually imagine that it will be very important to add any new reactions here for the success of the system.

Reply:
Ok, I will use enum data structure to represent post reaction. Make it simple and clear.